cmd/tfengine/README.md

```diff
@@ -119,7 +119,7 @@ terraform init
 terraform plan
 terraform apply
 
-# Step 6 (Optional): Deploy Continuous Integration (CI) and Continuous
+# Step 7 (Optional): Deploy Continuous Integration (CI) and Continuous
```

Comment:
seems should still be Step 6?

Reply:
I forgot to add the new step 6 :D

Reply:
Since the secrets are in live/ now. Maybe move it after CICD.

Reply:
Agreed